cmd/acra-keys/keys/keystore.go

```diff
@@ -245,3 +245,27 @@ func openKeyStoreV2(params KeyStoreParameters, loader keyloader.MasterKeyLoader)
 	}
 	return keystoreV2.NewServerKeyStore(keyDirectory), nil
 }
+
+// IsKeyStoreV2 checks if the directory contains a keystore version 2 from KeyStoreParameters
+func IsKeyStoreV2(params KeyStoreParameters) bool {
+	redisOption := params.RedisOptions()
+	if params.RedisConfigured() {
+		redisClient, err := filesystemBackendV2.OpenRedisBackend(&filesystemBackendV2.RedisConfig{
+			RootDir: params.KeyDir(),
+			Options: &redis.Options{
+				Addr:     redisOption.Addr,
+				Password: redisOption.Password,
+				DB:       redisOption.DB,
+			},
+		})
+		if err != nil {
+			log.WithError(err).Debug("Failed to find keystore v2 in Redis")
```

Comment:
```suggestion
			log.WithError(err).Debugln("Failed to find keystore v2 in Redis")
```
better to use appropriate Debug**ln** method for logging with new line at the end. because logrus may change API for such methods according to std `log` library that doesn't add \n at the end. Plus, we try to be compatible as much as possible  with standard library for case if we want to change logging backend.